was_modified.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import subprocess
+import sys
+import os.path
+
+# filepath starts from the root directory of the git project
+# with not trailing slash at the begining
+# the last -- it's no separte revisions from files
+files = subprocess.check_output(
+    ["git", "diff", "--name-only", "HEAD", "HEAD^1" , '--']).split()
```

Comment:
> Current problem is that I'm comparing HEAD to HEAD^1.
> But the you could push more than one commit at the time.

That's too dangerous, in a few months nobody will remember this pitfall. 

You could try `$TRAVIS_COMMIT_RANGE`, but that still has issues with history rewrites: https://github.com/travis-ci/travis-ci/issues/2668
One part of that problem might be solved using the Github API, see https://github.com/nest/nest-simulator/pull/75

Reply:
This is great! Thank you Nico! I did tried $TRAVIS_COMMIT_RANGE commit range and found the history rewrites problem the hard way.

Reply:
This seems to be another reasonable approach
https://github.com/reactor/reactor-core/pull/655/files